src/multiplots.ts

```diff
@@ -1266,14 +1248,17 @@ export class MultiplePlots {
       let isDrawingRubberObjIndex = this.get_drawing_rubberbands_obj_index('parallelplot');
       if (isDrawingRubberObjIndex != -1) {
         let isDrawingPP = this.objectList[isDrawingRubberObjIndex];
-        let rubberbands_dep = isDrawingPP.rubberbands_dep;
-        this.pp_communication(rubberbands_dep, isDrawingPP);
+        this.pp_communication(isDrawingPP.rubber_bands, isDrawingPP);
       }
     }
 
     pp_communication(rubberBands: RubberBand[], actualPP: any) { //process received data from a parallelplot and send it to the other objects
       let selectedIndices = actualPP.getObjectsInRubberBands(rubberBands);
-
+      let rubberBandNames = []
+      rubberBands.forEach((rubberBand) => {
```

Comment:
could take advantage of the `map` syntax istead of `forEach` : 

`const rubberBandNames = rubberBands.forEach((rubberBand) => rubberBand.attributeName);`

Reply:
This does not work. I think I must instantiate an empty array before. If not, i did not could you give me the way to do it please ?